pkg/service/replay/utils.go

```diff
@@ -130,6 +140,421 @@ func (t *requestMockUtil) ProcessMockFile(_ context.Context, testSetID string) {
 	t.logger.Debug("Mock file for test set", zap.String("testSetID", testSetID))
 }
 
+// Parse the json string into a geko type variable.
```

Comment:
can you move the entire template-related logic to a separate file called `template.go` for now?